install-dev/data/db_structure.sql

```diff
@@ -17,7 +17,6 @@ CREATE TABLE `PREFIX_address` (
   `id_customer` int(10) unsigned NOT NULL DEFAULT '0',
   `id_manufacturer` int(10) unsigned NOT NULL DEFAULT '0',
   `id_supplier` int(10) unsigned NOT NULL DEFAULT '0',
```

Comment:
Could you create migration SQL for removing these columns ?

Reply:
I'm unsure if it's a good idea to delete data or not

Reply:
Since some modules rely on this data I'd say we shouldn't remove the column during upgrade This way the used data won't be lost
Of course for the version >=8 the module will need to add the column itself